### DISCUSSION TITLE
I don't know where to install the themes.

### DISCUSSION THREAD

**Discussion by samcastr0** [ohmybash/oh-my-bash#521] [Q&A]
Guys, good afternoon!
I'm speaking from Brazil here and I need your help to install the themes.

I am currently using **LINUX MINT LMDE 6, 64-bit system.**

I'm new to the Linux world, I migrated 2 months ago and I'm still catching on a bit.

Can anyone help me find the bash themes folder?

Hug!

**Comment by akinomyoga** [comment]
> I'm speaking from Brazil here and I need your help to install the themes.

Have you installed Oh My Bash (OMB)? To use those OMB themes, you need to install OMB. Once OMB is installed, you don't have to do anything additional to use the themes.

> Can anyone help me find the bash themes folder?

What is that? I think there is no such thing in Bash, but could you describe what is your assumption about how "*the bash themes folder*" works? Literally reading, I think the closest is [this](https://github.com/ohmybash/oh-my-bash/tree/master/themes), but I guess that's not what you are asking.

**Reply by samcastr0** [reply]
I installed **OH MY BASH** and thought I would have to download these files when in fact I saw that they are already there but I don't know how to change the theme.

Can you help me?

**Reply by ariedallas** [reply]
You have to go into your .bashrc file in your home directory. There should be a line in that file with the word 'theme' in it.

**Comment by xenodesire** [comment]
> Guys, good afternoon! I'm speaking from Brazil here and I need your help to install the themes.
> 
> I am currently using **LINUX MINT LMDE 6, 64-bit system.**
> 
> I'm new to the Linux world, I migrated 2 months ago and I'm still catching on a bit.
> 
> Can anyone help me find the bash themes folder?
> 
> Hug!

Também sou brasileiro, a pasta de temas esta em ~/.oh-my-bash/themes

**Comment by neslob** [comment]
It is in your `.bashrc` and it is called `OSH_THEME`.

Set it to `OSH_THEME="vscode"`. Then `source .bashrc` to see the changes. 

One liner `NEWTHEME="vscode" && sed -i "s/^OSH_THEME=.*/OSH_THEME=\"$NEWTHEME\"/" ~/.bashrc && source ~/.bashrc` .
